docs/source/_json/content_get.resp

```diff
@@ -49,5 +49,6 @@ Content-Type: application/json
   "table_of_contents": null, 
   "text": null, 
   "title": "My Document", 
-  "version": "current"
+  "version": "current", 
+  "versioning_enabled": true
```

Comment:
I'm a bit confused about why these changes in the req/resp files happen here, even though this PR
- doesn't bump the Plone version
- doesn't remove the `skipTest()` from `test_documentation.py`

Reply:
I was running it with plone 5.1 - that explains it perhaps. 
Currently builds 4 and 5.1 succeeds, while 5.0 fails. (because of documentation artifacts)

Reply:
Ah, righ - that's what I would have expected to happen. I would recommend to keep 5.0 as the reference-build for the documentation artifacts (that's defined [here](https://github.com/plone/plone.restapi/blob/master/test-no-uncommitted-doc-changes.in#L16-L21), though the comment didn't get updated when the reference build got switched from 4.3 to 5.0).

I would address actually moving to 5.1 (for the default buildout, and the reference build for docs artifacts, and updating them / checking changes) in a separate PR.